doc/CMakeLists.txt

```diff
@@ -488,3 +488,63 @@ add_custom_target(
   COMMAND ${CMAKE_MAKE_PROGRAM} nrf
   COMMAND ${CMAKE_MAKE_PROGRAM} nrfxlib
 )
+
+
+#
+# Add targets for checking links
+#
+
+add_custom_target(
+  nrf-linkcheck
+  COMMAND ${CMAKE_COMMAND} -E env
+  ZEPHYR_BASE=${ZEPHYR_BASE}
+  ZEPHYR_BUILD=${ZEPHYR_BINARY_DIR}
+  ZEPHYR_OUTPUT=${HTML_DIR}/zephyr
+  NRF_BASE=${NRF_BASE}
+  NRF_BUILD=${NRF_BINARY_DIR}
+  NRF_OUTPUT=${HTML_DIR}/nrf
+  KCONFIG_OUTPUT=${HTML_DIR}/kconfig
+  NRF_RST_SRC=${NRF_RST_OUT}/doc/nrf
+  ZEPHYR_RST_SRC=${ZEPHYR_BINARY_DIR}/rst/doc
+  MCUBOOT_OUTPUT=${HTML_DIR}/mcuboot
+  NRFXLIB_OUTPUT=${HTML_DIR}/nrfxlib
```

Comment:
Those are use multiple time, could you make a common setting and just reuse, like this:
```
set(sphinx_environment 
    ZEPHYR_BASE=${ZEPHYR_BASE}
    ZEPHYR_BUILD=${ZEPHYR_BINARY_DIR}
    ZEPHYR_OUTPUT=${HTML_DIR}/zephyr
    NRF_BASE=${NRF_BASE}
    NRF_BUILD=${NRF_BINARY_DIR}
    NRF_OUTPUT=${HTML_DIR}/nrf
    KCONFIG_OUTPUT=${HTML_DIR}/kconfig
    NRF_RST_SRC=${NRF_RST_OUT}/doc/nrf
    ZEPHYR_RST_SRC=${ZEPHYR_BINARY_DIR}/rst/doc
    MCUBOOT_OUTPUT=${HTML_DIR}/mcuboot
    NRFXLIB_OUTPUT=${HTML_DIR}/nrfxlib
)

add_custom_target(
  <target>
  COMMAND ${CMAKE_COMMAND} -E env ${sphinx_environment}
  .... rest of command
  COMMAND_EXPAND_LISTS
  ...
)

Reply:
Thanks - updated. :)